app/controllers/webhooks_controller.rb

```diff
@@ -0,0 +1,27 @@
+class WebhooksController < ApplicationController
+  include ShopifyApp::WebhookVerification
+
+  class ShopifyApp::MissingWebhookJobError < StandardError; end
+
+  def receive
+    job_args = {shop_domain: shop_domain, webhook: webhook_params}
+    webhook_job_klass.perform_later(job_args)
+    head :ok
```

Comment:
You are returning `head :ok` from each webhook instead of `head :no_content`. I don't think there is a functional difference (200 response vs 204), but I’d check just in case.